packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -881,6 +881,7 @@ abstract class BaseFlutterTask extends DefaultTask {
             ruleNames = targetPlatformValues.collect { "android_aot_bundle_${buildMode}_$it" }
         }
         project.exec {
+            logging.captureStandardError LogLevel.ERROR
```

Comment:
should we update or add a new integration test?

Reply:
Yes we should, easy to miss this one.

Reply:
Added a test, as verifies the output message